nucleus/deploy/client.py

```diff
@@ -103,27 +103,32 @@ def register_endpoint_auth_decorator(self, endpoint_auth_decorator_fn):
     def create_model_bundle(
         self,
         model_bundle_name: str,
-        load_predict_fn: Callable[[DeployModel_T], Callable[[Any], Any]],
         env_params: Dict[str, str],
+        *,
+        load_predict_fn: Optional[
+            Callable[[DeployModel_T], Callable[[Any], Any]]
+        ] = None,
+        predict_fn_or_cls: Optional[Callable[[Any], Any]] = None,
```

Comment:
Overall I think this is fine for now, but we might as well start deprecating `load_predict_fn` later.

Reply:
Oh I guess same comment as https://github.com/scaleapi/models/pull/2684/files#r820019055, basically.